### DISCUSSION TITLE
Set raw event body limit in environment variables

### DISCUSSION THREAD

**Discussion by StefjJHK** [datalust/seq-tickets#1541] [Help]
I need to log big body. Default value is set to  256 KB, how can I set my custom value in environment variables to use it in docker-compose?
![image](https://user-images.githubusercontent.com/66734934/161976296-cc788ede-03c9-4a65-b13d-4adbdeb0d113.png)

**Comment by liammclennan** [comment]
Hi @StefjJHK ,

The maximum event size is set via the Seq API, not by an environment variable. You should be able to use the API if you need to automate changing the maximum event size. The endpoint is `/api/settings/setting-raweventmaximumcontentlength`. Open your browser developer tools, change the value, save it and you will see the format of the request. 

As noted above, changing this value will reduce the performance of your server.
